lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -353,17 +347,30 @@ defmodule RealtimeWeb.RealtimeChannel do
   def handle_in("presence", payload, %{assigns: %{private?: true}} = socket) do
     %{tenant: tenant_id} = socket.assigns
 
-    with {:ok, db_conn} <- Connect.lookup_or_start_connection(tenant_id) do
-      PresenceHandler.handle(payload, db_conn, socket)
+    with {:ok, db_conn} <- Connect.lookup_or_start_connection(tenant_id),
+         {:ok, socket} <- PresenceHandler.handle(payload, db_conn, socket) do
+      {:noreply, socket}
```

Comment:
So do we want to change `track` to not reply ok here? Can this impact a client code that expects this? 🤔